### PR TITLE
Show more messages on other section

### LINKED ISSUE #7659: *Other* message in my inbox not visible without scrolling
### Steps to reproduce

0. Have a 1920x1080 screen
1. Open the app
2. Add an account
3. Open the priority inbox

### Expected behavior

I see the latest message(s) in *Other*

### Actual behavior

I only see important messages

Bildschirmfoto vom 2022-11-23 09-19-36

### Mail app version

1.14

### Mailserver or service

_No response_

### Operating system

_No response_

### PHP engine version

_No response_

### Web server

_No response_

### Database

_No response_

### Additional info

Regression of https://github.com/nextcloud/mail/issues/6142

@jancborchardt @nimishavijay is this intended?

### PR DESCRIPTION
**Pull request by GretaD**
fixes #7659

1920X1080 screensize
Screenshot from 2022-11-28 10-38-12
mobile
Screenshot from 2022-11-28 10-34-04

### CODE REVIEW

**Review by ChristophWurst — COMMENTED** [review]
Shouldn't this be some kind of dependent on the screen size?

**Comment by GretaD** [comment]
> Shouldn't this be some kind of dependent on the screen size?

the envelope list already depends on the screen size on how many envelopes will be shown in total. What this pr does(or the old one did) is that "forces" 6 messages on the important section by default, before you click load more.

File: src/components/Mailbox.vue [hunk 1]
```diff
@@ -129,7 +129,7 @@ export default {
 		},
 		envelopesToShow() {
 			if (this.paginate === 'manual' && !this.expanded) {
-				return this.envelopes.slice(0, 8)
+				return this.envelopes.slice(0, 6)
```
Comment (ChristophWurst) [inline src/components/Mailbox.vue:132]: A bit smelly that you have to adjust the same magic number in two paces, isn't it? Replace this occurrence by `this.initialPageSize`

Comment (GretaD) [reply src/components/Mailbox.vue:132]: > Replace this occurrence by this.initialPageSize

thats true, will do. But i still dont see how i can change the number on envelopes shown on important section depending on the screen, when we have a initialization of 6 envelopes and a 'manual' pagination/loadmore

Comment (ChristophWurst) [reply src/components/Mailbox.vue]: Derive a guesstimate from `window.innerHeight`

**Review by jancborchardt — COMMENTED** [review]
Sounds good if indeed it works as designed and always shows a bit of "Other" at the bottom.

**Comment by GretaD** [comment]
> Sounds good if indeed it works as designed and always shows a bit of "Other" at the bottom.

then please approve :)

**Review by jancborchardt — APPROVED** [review]
Aye aye! :D

**Pull request merged**